lib/traject/config/folio_resource_type.rb

```diff
@@ -0,0 +1,140 @@
+# frozen_string_literal: true
+
+require_relative '../macros/resource_type'
+
+module FolioResourceType
+  include Traject::Macros::ResourceType
+
+  # rubocop:disable Metrics/AbcSize
+  # rubocop:disable Metrics/MethodLength
+  def add_folio_resource_type_fields
+    to_field 'resource_type_ssim',
```

Comment:
This is going to drive a new facet field called "format", and it looks like our hierarchical call number field uses the suffix "hsim", so maybe something like 'format_hsim' is a better name? 

```suggestion
    to_field 'format_hsim',
```